framework/src/endpoint.ts

```diff
@@ -12,16 +12,28 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import { BaseEndpoint } from './modules';
+import { DynamicRewardEndpoint } from './modules/dynamic_rewards/endpoint';
```

Comment:
Framework in general should not be aware of any module.
If we need specific handling, we need to consider different approach in the module side